### PR TITLE
[GH Issue Template] Add issue template for mesheryctl and edit mesheryctl README

### LINKED ISSUE #2565: [GH Issue Template] Create new GitHub issue template for mesheryctl
#### Current Behavior
See other templates as examples.

#### Desired Behavior
1. Prominent reference - link to command spreadsheet.
1. Resources - Include links to contributor guide, tutorial on contributing to mesheryctl.
2. label with "component/mesheryctl"

### PR DESCRIPTION
**Pull request by pottekkat**
Signed-off-by: navendu-pottekkat <navendupottekkat@gmail.com>

**Description**

This PR fixes #2565 

Added a new issue template for `mesheryctl` which has references to all the contributor resources and [mesheryctl- Command Reference and Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0) document.

- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by aneeshnema — DISMISSED** [review]
LGTM 💯

File: .github/ISSUE_TEMPLATE/component_mesheryctl.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+name: ⌨️ Component- mesheryctl(Meshery CLI)
+about: Report an issue/ faeture request for mesheryctl(Meshery CLI)
+title: '[mesheryctl]'
+labels: 'component/mesheryctl'
+assignees: ''
+---
+
+<!-- Please update the mesheryctl- Command Reference and Tracker spreadsheet -->
+Refer the [mesheryctl- Command Reference and Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0) for current status of `mesheryctl`.
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/component_mesheryctl.md:10]: No need for the hyphen after mesheryctl

File: .github/ISSUE_TEMPLATE/component_mesheryctl.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Kind: [Bug/Feature Request]
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/component_mesheryctl.md:12]: Remove this line, please.

File: .github/ISSUE_TEMPLATE/component_mesheryctl.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+name: ⌨️ Component- mesheryctl(Meshery CLI)
+about: Report an issue/ faeture request for mesheryctl(Meshery CLI)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/component_mesheryctl.md:3]: Feature misspelled

File: .github/ISSUE_TEMPLATE/component_mesheryctl.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+---
+name: ⌨️ Component- mesheryctl(Meshery CLI)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/component_mesheryctl.md:2]: Space missing after mesheryctl. 


Use colon, not hyphen after Component.

File: .github/ISSUE_TEMPLATE/component_mesheryctl.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+#### Mockups
+<!-- Any visual diagrams of the desired user interface. -->
+
+---
+
+#### Contributor Resources
+
+- [mesheryctl Contributing Guide](https://github.com/layer5io/meshery/blob/master/mesheryctl/README.md)
+- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
+- [mesheryctl- Command Reference and Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/component_mesheryctl.md:53]:
```suggestion
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
```

File: .github/ISSUE_TEMPLATE/component_mesheryctl.md (hunk 1 above)
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/component_mesheryctl.md:10]:
```suggestion
Refer the [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0) for current status of `mesheryctl`.
```

**Comment by navendu-pottekkat** [comment]
@leecalcote made the changes as you mentioned. We can merge this.

**Pull request merged**